src/Pelagos/Bundle/AppBundle/Resources/views/MdApp/upload-complete.html.twig

```diff
@@ -2,6 +2,8 @@
 
 {% block head %}
     {{ parent() }}
+    
```

Comment:
The upload from MDAPP behaviour has been replaced by the dataset-review process. At some point we should probably refactor and remove these vestigial stubs.